app/build.gradle.kts

```diff
@@ -16,6 +16,7 @@ android {
         versionName = ProjectConfig.versionName
 
         testInstrumentationRunner = "com.plcoding.calorytracker.HiltTestRunner"
+        testInstrumentationRunnerArguments["clearPackageData"] = "true"
```

Comment:
I set to `true`,  the Android Gradle plugin will clear all app data on the device before each test. This ensures a clean environment for each test, avoiding potential side-effects from previous test runs or manual usage of the app.
Setting clearPackageData to true can make tests more reliable at the cost of increased execution time.

https://developer.android.com/training/testing/instrumented-tests/androidx-test-libraries/runner#use-android